# Exploration-of-Basic-Models-in-Tumor-Related-Fields
This article summarizes a series of research progress on basic models in the field of cancer, integrating the current mainstream artificial intelligence basic models used in the diagnosis and treatment of tumors.
# Paper List
## Modeling framework
### CNN-based method for assisted tumor diagnosis and treatment
1.Fast, E., Dhar, M., & Chen, B. (2023). TAPIR: a T-cell receptor language model for predicting rare and novel targets. bioRxiv : the preprint server for biology, 2023.09.12.557285. https://doi.org/10.1101/2023.09.12.557285

2.Gao, Yang et al. “An integrated model incorporating deep learning, hand-crafted radiomics and clinical and US features to diagnose central lymph node metastasis in patients with papillary thyroid cancer.” BMC cancer vol. 24,1 69. 12 Jan. 2024, doi:10.1186/s12885-024-11838-1

3.Le, N. Q. K., & Huynh, T. T. (2019). Identifying SNAREs by Incorporating Deep Learning Architecture and Amino Acid Embedding Representation. Frontiers in physiology, 10, 1501. https://doi.org/10.3389/fphys.2019.01501

4.Lee, H., Lee, Y., Jung, S. W., Lee, S., Oh, B., & Yang, S. (2023). Deep Learning-Based Evaluation of Ultrasound Images for Benign Skin Tumors. Sensors (Basel, Switzerland), 23(17), 7374. https://doi.org/10.3390/s23177374  

5.Malusare, A., Kothandaraman, H., Tamboli, D., Lanman, N. A., & Aggarwal, V. (2024). Understanding the natural language of DNA using encoder-decoder foundation models with byte-level precision. Bioinformatics advances, 4(1), vbae117. https://doi.org/10.1093/bioadv/vbae117

6.Yao, Lantian et al. “ACP-CapsPred: an explainable computational framework for identification and functional prediction of anticancer peptides based on capsule network.” Briefings in bioinformatics vol. 25,5 (2024): bbae460. doi:10.1093/bib/bbae460

7.Shahid Malik, M., & Ou, Y. Y. (2023). Integrating Pre-Trained protein language model and multiple window scanning deep learning networks for accurate identification of secondary active transporters in membrane proteins. Methods (San Diego, Calif.), 220, 11–20. https://doi.org/10.1016/j.ymeth.2023.10.008
### Transformer-based method for assisted tumor diagnosis and treatment
1.Hua, S., Yan, F., Shen, T., Ma, L., & Zhang, X. (2024). PathoDuet: Foundation models for pathological slide analysis of H&E and IHC stains. Medical image analysis, 97, 103289. https://doi.org/10.1016/j.media.2024.103289

2.Vorontsov, E., Bozkurt, A., Casson, A., Shaikovski, G., Zelechowski, M., Severson, K., Zimmermann, E., Hall, J., Tenenholtz, N., Fusi, N., Yang, E., Mathieu, P., van Eck, A., Lee, D., Viret, J., Robert, E., Wang, Y. K., Kunz, J. D., Lee, M. C. H., Bernhard, J. H., … Fuchs, T. J. (2024). A foundation model for clinical-grade computational pathology and rare cancers detection. Nature medicine, 30(10), 2924–2935. https://doi.org/10.1038/s41591-024-03141-0

3.Cox, J., Liu, P., Stolte, S. E., Yang, Y., Liu, K., See, K. B., Ju, H., & Fang, R. (2024). BrainSegFounder: Towards 3D foundation models for neuroimage segmentation. Medical image analysis, 97, 103301. https://doi.org/10.1016/j.media.2024.103301

4.Chen, M., Zhang, M., Yin, L., Ma, L., Ding, R., Zheng, T., Yue, Q., Lui, S., & Sun, H. (2024). Medical image foundation models in assisting diagnosis of brain tumors: a pilot study. European radiology, 34(10), 6667–6679. https://doi.org/10.1007/s00330-024-10728-1

5.Kwak, K. M., Choi, G. H., Shim, K. E., Jin, H. Y., Kim, S. H., Yoon, J. W., & Kim, H. (2020). Liver Infarction and Venous Thromboembolism after Tamoxifen Use in an ADPKD Patient with Encapsulating Peritoneal Sclerosis: A Case Report. Electrolyte & blood pressure : E & BP, 18(2), 44–48. https://doi.org/10.5049/EBP.2020.18.2.44

6.Dehghani, N., Saliba-Colombani, V., Chick, A., Heng, M., Operto, G., & Fillard, P. (2024). Efficient labeling of french mammogram reports with MammoBERT. Scientific reports, 14(1), 24842. https://doi.org/10.1038/s41598-024-76369-y

7.Elsamahy, E. A., Ahmed, A. E., Shoala, T., & Maghraby, F. A. (2024). Deep-GenMut: Automated genetic mutation classification in oncology: A deep learning comparative study. Heliyon, 10(11), e32279. https://doi.org/10.1016/j.heliyon.2024.e32279

8.okinen, E., Dumitrescu, A., Huuhtanen, J., Gligorijević, V., Mustjoki, S., Bonneau, R., Heinonen, M., & Lähdesmäki, H. (2023). TCRconv: predicting recognition between T cell receptors and epitopes using contextualized motifs. Bioinformatics (Oxford, England), 39(1), btac788. https://doi.org/10.1093/bioinformatics/btac788

9.Kwak, K. M., Choi, G. H., Shim, K. E., Jin, H. Y., Kim, S. H., Yoon, J. W., & Kim, H. (2020). Liver Infarction and Venous Thromboembolism after Tamoxifen Use in an ADPKD Patient with Encapsulating Peritoneal Sclerosis: A Case Report. Electrolyte & blood pressure : E & BP, 18(2), 44–48. https://doi.org/10.5049/EBP.2020.18.2.44

10.Choi Y. (2022). Artificial intelligence for antibody reading comprehension: AntiBERTa. Patterns (New York, N.Y.), 3(7), 100535. https://doi.org/10.1016/j.patter.2022.100535

11.Sanabria, M., Hirsch, J., & Poetsch, A. R. (2024). Distinguishing word identity and sequence context in DNA language models. BMC bioinformatics, 25(1), 301. https://doi.org/10.1186/s12859-024-05869-5

12。Brixi, G., Ye, T., Hong, L., Wang, T., Monticello, C., Lopez-Barbosa, N., Vincoff, S., Yudistyra, V., Zhao, L., Haarer, E., Chen, T., Pertsemlidis, S., Palepu, K., Bhat, S., Christopher, J., Li, X., Liu, T., Zhang, S., Petersen, L., DeLisa, M. P., … Chatterjee, P. (2023). SaLT&PepPr is an interface-predicting language model for designing peptide-guided protein degraders. Communications biology, 6(1), 1081. https://doi.org/10.1038/s42003-023-05464-z

13.Chandra, A., Sharma, A., Dehzangi, I., Tsunoda, T., & Sattar, A. (2023). PepCNN deep learning tool for predicting peptide binding residues in proteins using sequence, structural, and language model features. Scientific reports, 13(1), 20882. https://doi.org/10.1038/s41598-023-47624-5  

14.Le, V. T., Tseng, Y. H., Liu, Y. C., Malik, M. S., & Ou, Y. Y. (2024). VesiMCNN: Using pre-trained protein language models and multiple window scanning convolutional neural networks to identify vesicular transport proteins. International journal of biological macromolecules, 280(Pt 3), 136048. Advance online publication. https://doi.org/10.1016/j.ijbiomac.2024.136048

15.Bhat, S., Palepu, K., Hong, L., Mao, J., Ye, T., Iyer, R., Zhao, L., Chen, T., Vincoff, S., Watson, R., Wang, T., Srijay, D., Kavirayuni, V. S., Kholina, K., Goel, S., Vure, P., Desphande, A. J., Soderling, S. H., DeLisa, M. P., & Chatterjee, P. (2024). De Novo Design of Peptide Binders to Conformationally Diverse Targets with Contrastive Language Modeling. bioRxiv : the preprint server for biology, 2023.06.26.546591. https://doi.org/10.1101/2023.06.26.546591  

16.Sherpa, P., Chong, K. T., & Tayara, H. (2024). FvFold: A model to predict antibody Fv structure using protein language model with residual network and Rosetta minimization. Computers in biology and medicine, 182, 109128. https://doi.org/10.1016/j.compbiomed.2024.109128
Zhang, M., Cheng, Q., Wei, Z., Xu, J., Wu, S., Xu, N., Zhao, C., Yu, L., & Feng, W. (2024). BertTCR: a Bert-based deep learning framework for predicting cancer-related immune status based on T cell receptor repertoire. Briefings in bioinformatics, 25(5), bbae420. https://doi.org/10.1093/bib/bbae420

17.Sorin, V., Klang, E., Sklair-Levy, M., Cohen, I., Zippel, D. B., Balint Lahat, N., Konen, E., & Barash, Y. (2023). Large language model (ChatGPT) as a support tool for breast tumor board. NPJ breast cancer, 9(1), 44. https://doi.org/10.1038/s41523-023-00557-8  

18.Yeo, Y. H., Samaan, J. S., Ng, W. H., Ting, P. S., Trivedi, H., Vipani, A., Ayoub, W., Yang, J. D., Liran, O., Spiegel, B., & Kuo, A. (2023). Assessing the performance of ChatGPT in answering questions regarding cirrhosis and hepatocellular carcinoma. Clinical and molecular hepatology, 29(3), 721–732. https://doi.org/10.3350/cmh.2023.0089

19.Sorin, V., Klang, E., Sklair-Levy, M., Cohen, I., Zippel, D. B., Balint Lahat, N., Konen, E., & Barash, Y. (2023). Large language model (ChatGPT) as a support tool for breast tumor board. NPJ breast cancer, 9(1), 44. https://doi.org/10.1038/s41523-023-00557-8
[57]Lukac, S., Dayan, D., Fink, V., Leinert, E., Hartkopf, A., Veselinovic, K., Janni, W., Rack, B., Pfister, K., Heitmeir, B., & Ebner, F. (2023). Evaluating ChatGPT as an adjunct for the multidisciplinary tumor board decision-making in primary breast cancer cases. Archives of gynecology and obstetrics, 308(6), 1831–1844. https://doi.org/10.1007/s00404-023-07130-5
[58]Huang, Y., Gomaa, A., Semrau, S., Haderlein, M., Lettmaier, S., Weissmann, T., Grigo, J., Tkhayat, H. B., Frey, B., Gaipl, U., Distel, L., Maier, A., Fietkau, R., Bert, C., & Putz, F. (2023). Benchmarking ChatGPT-4 on a radiation oncology in-training exam and Red Journal Gray Zone cases: potentials and challenges for ai-assisted medical education and decision making in radiation oncology. Frontiers in oncology, 13, 1265024. https://doi.org/10.3389/fonc.2023.1265024
[59]Hung, T. K. W., Kuperman, G. J., Sherman, E. J., Ho, A. L., Weng, C., Pfister, D. G., & Mao, J. J. (2024). Performance of Retrieval-Augmented Large Language Models to Recommend Head and Neck Cancer Clinical Trials. Journal of medical Internet research, 26, e60695. https://doi.org/10.2196/60695
[60]Hegde, A., Srinivasan, S., & Menon, G. (2023). Extraventricular Neurocytoma of the Posterior Fossa: A Case Report Written by ChatGPT. Cureus, 15(3), e35850. https://doi.org/10.7759/cureus.35850
[61]Rao, A., Kim, J., Kamineni, M., Pang, M., Lie, W., & Succi, M. D. (2023). Evaluating ChatGPT as an Adjunct for Radiologic Decision-Making. medRxiv : the preprint server for health sciences, 2023.02.02.23285399. https://doi.org/10.1101/2023.02.02.23285399
[62]Huang, J., Yang, D. M., Rong, R., Nezafati, K., Treager, C., Chi, Z., Wang, S., Cheng, X., Guo, Y., Klesse, L. J., Xiao, G., Peterson, E. D., Zhan, X., & Xie, Y. (2024). A critical assessment of using ChatGPT for extracting structured data from clinical notes. NPJ digital medicine, 7(1), 106. https://doi.org/10.1038/s41746-024-01079-8
[63]Sultan, I., Al-Abdallat, H., Alnajjar, Z., Ismail, L., Abukhashabeh, R., Bitar, L., & Abu Shanap, M. (2023). Using ChatGPT to Predict Cancer Predisposition Genes: A Promising Tool for Pediatric Oncologists. Cureus, 15(10), e47594. https://doi.org/10.7759/cureus.47594
[64]Wang, A., Zhou, J., Zhang, P., Cao, H., Xin, H., Xu, X., & Zhou, H. (2024). Large language model answers medical questions about standard pathology reports. Frontiers in medicine, 11, 1402457. https://doi.org/10.3389/fmed.2024.1402457  
[65]Liu, S., Pastor-Serrano, O., Chen, Y., Gopaulchan, M., Liang, W., Buyyounouski, M., Pollom, E., Le, Q. T., Gensheimer, M., Dong, P., Yang, Y., Zou, J., & Xing, L. (2024). Automated radiotherapy treatment planning guided by GPT-4Vision. ArXiv, arXiv:2406.15609v2.
[66]Peng, C., Yang, X., Chen, A., Yu, Z., Smith, K. E., Costa, A. B., Flores, M. G., Bian, J., & Wu, Y. (2024). Generative large language models are all-purpose text analytics engines: text-to-text learning is all your need. Journal of the American Medical Informatics Association : JAMIA, 31(9), 1892–1903. https://doi.org/10.1093/jamia/ocae078
[67]Bhayana, R., Nanda, B., Dehkharghanian, T., Deng, Y., Bhambra, N., Elias, G., Datta, D., Kambadakone, A., Shwaartz, C. G., Moulton, C. A., Henault, D., Gallinger, S., & Krishna, S. (2024). Large Language Models for Automated Synoptic Reports and Resectability Categorization in Pancreatic Cancer. Radiology, 311(3), e233117. https://doi.org/10.1148/radiol.233117
[68]Mitsuyama, Y., Tatekawa, H., Takita, H., Sasaki, F., Tashiro, A., Oue, S., Walston, S. L., Nonomiya, Y., Shintani, A., Miki, Y., & Ueda, D. (2025). Comparative analysis of GPT-4-based ChatGPT's diagnostic performance with radiologists using real-world radiology reports of brain tumors. European radiology, 35(4), 1938–1947. https://doi.org/10.1007/s00330-024-11032-8
